src/main/webapp/js/common.es6

```diff
@@ -988,7 +988,7 @@ function setChevronToDown(chevronContainer) {
 function toggleChevron(clickedElement) {
     const $clickedElement = $(clickedElement);
     const isChevronDown = $clickedElement.find('.glyphicon-chevron-down').length > 0;
-    const $chevronContainer = $clickedElement.find('.glyphicon');
+    const $chevronContainer = $clickedElement.find('.glyphicon-chevron-up, .glyphicon-chevron-down');
```

Comment:
Let's assume this is the sample code:
``` HTML
<div class="parent" onclick="toggleChevron(this)">
    <span class="glyphicon glyphicon-chevron-down">
    <span class="glyphicon glyphicon-search">
</div>
```
As you can see we have two element with class name `glyphicon` inside the parent `div`. Everytime the user clicks on the parent div, the `toggleChevron` function will be called and it will look for all elements with class name `glyphicon` inside the parent div and will toggle classes of thoes elements with class name `glyphicon`.

**_Solution:_**
Instead of looking for elements with class name `glyphicon`, we should look for elements with the class name either `glyphicon-chevron-up` or `glyphicon-chevron-down`.